src/evaluations/tests/analyzer_test.py

```diff
@@ -258,6 +258,59 @@ def test_get_analysis_results(self):
 
 class FrequencyEstimatorEvaluationAnalyzerTest(absltest.TestCase):
 
+  def setUp(self):
+    super(FrequencyEstimatorEvaluationAnalyzerTest, self).setUp()
+
+    self.sketch_estimator_config_list = [
+        configs.SketchEstimatorConfig(
+            name=evaluation_configs.construct_sketch_estimator_config_name(
+                sketch_name='exact_multi_set',
+                sketch_config='10000',
+                estimator_name='lossless'),
+            sketch_factory=exact_set.ExactMultiSet.get_sketch_factory(),
+            estimator=exact_set.LosslessEstimator(),
+            max_frequency=2,
+        ),
+    ]
+
+    self.evaluation_config = configs.EvaluationConfig(
+        name='frequency_end_to_end_test',
+        num_runs=1,
+        scenario_config_list=[
+            configs.ScenarioConfig(
+                name='homogeneous',
+                set_generator_factory=(
+                    frequency_set_generator.HomogeneousMultiSetGenerator
+                    .get_generator_factory_with_num_and_size(
+                        universe_size=100,
+                        num_sets=3,
+                        set_size=50,
+                        freq_rates=[5] * 3,
+                        freq_cap=8,
+                    )))]
+    )
+
+    self.run_name = 'test_run'
+
+    def _get_test_evaluator(out_dir):
```

Comment:
Don't intersperse helper methods in code. Either make module level or put at the top of the function.

Reply:
Done.